src/index.test.ts

```diff
@@ -1,5 +1,6 @@
-import { functionValidator, validator, Context, Event } from './index';
```

Comment:
@ahcai no idea why the linter `npm run lint:fix` decided to do this. But I'll apply twilio-style in another PR and use that linter instead.